dotcom-rendering/cdk/bin/cdk.ts

```diff
@@ -4,10 +4,12 @@ import { DotcomRendering } from '../lib/dotcom-rendering';
 
 const app = new App();
 new DotcomRendering(app, 'DotcomRendering-PROD', {
+	app: 'rendering',
```

Comment:
From what [I can see in the logs](https://logs.gutools.co.uk/s/dotcom/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(columns:!(),filters:!(),index:'50201630-bd0f-11e9-92b3-f10623802d36',interval:auto,query:(language:kuery,query:'app.keyword%20:%20%22dotcom-rendering%22%20'),sort:!(!('@timestamp',desc)))), this is actually called `dotcom-rendering`.

Or is this set elsewhere?

Reply:
Our "naming" is REALLY inconsistent, but I believe "rendering" is correct. Its set here: https://github.com/guardian/dotcom-rendering/pull/8428/files#diff-27418f23c8cbec59b8753ad54c095fd59254f5626f1a48ce77acd161fb3ab3c7R17

`AWS Resource name =/= ELK App name` unfortunately :/